glomentorprogram/src/components/SearchForm/SearchForm.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import './SearchForm.css';
+
+function SearchForm({ initialSearchQuery, onSearch }) {
+	const [searchQuery, setSearchQuery] = useState(initialSearchQuery);
```

Comment:
@bcvitanusic although this is working one of the requirements is to pass the initial value from props. You are using the `initialSearchQuery` in the placeholder property instead to use into `defaultValue` property.

Take a [look here](https://react.dev/reference/react-dom/components/input#providing-an-initial-value-for-an-input) to check in details

Reply:
fixed